docs/Installation/Installation_Guide.rst

```diff
@@ -492,29 +492,28 @@ parameters as appropriate.)
    ::
 
     ingress:
-     enabled: "true"
-     annotations:
-       nginx.ingress.kubernetes.io/proxy-body-size: 0
-
-    hosts:
-      -  zenko.local
+      enabled: "true"

```

Comment:
I see indentation levels in this file as both a single and double space. For example: [here](https://github.com/scality/Zenko/pull/268/files#diff-c22da790284d7726aa094bb10c9def89R214) it is still single space indentation.

Reply:
Yeah, I don't understand why. The metalk8s section will be removed anyways.